bundle/run/job_options.go

```diff
@@ -72,6 +77,21 @@ func (o *JobOptions) Validate(job *resources.Job) error {
 		return errors.New("the job to run does not define job parameters; specifying job parameters is not allowed")
 	}
 
+	if len(o.tasks) > 0 {
+		for _, task := range o.tasks {
+			found := false
+			for _, t := range job.Tasks {
+				if t.TaskKey == task {
+					found = true
+					break
+				}
+			}
+			if !found {
+				return fmt.Errorf("task %s not found in job %s", task, job.Name)
```

Comment:
nit: %v could be nicer in some cases, e.g. if there are empty strings involved due to user error.

Reply:
thanks, updated here https://github.com/databricks/cli/pull/3388/commits/1191074420ceabf89520641b95989d22b7ec9e29

Reply:
fixed here https://github.com/databricks/cli/pull/3388/commits/c9af526ad81f1b766cac41bdf729ca3098f87d1f